### PR TITLE
added filter by category

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following

1. Filter by a category
  - All
  - Trending
  - Smart Wearables
  - Smart Phones
  - Laptops
 
Preview Link: https://deploy-preview-9--videoporium.netlify.app/

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
code look neat, code sturcture is also good

File: src/components/Category/Category.jsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+  const { categoryDispatch, selectedCategory } = useCategory();
 
-  return <button class="button chip">{categoryName}</button>;
+  return <button className={`${
+    videoCategory === selectedCategory ? "selected" : ""
+  } button chip`} onClick={() =>
+    categoryDispatch({
+      type: "CATEGORY",
+      payload: videoCategory
+    })
```
Comment (rohanmathur91) [inline src/components/Category/Category.jsx]: suggestion: have another function say `handleClick` and inside that put all this and give `handleClick` to onClick

**Review by rohanmathur91 — APPROVED** [review]
- UI looks good
- Good file structure
- PR looks good to merge

File: src/components/Category/Category.jsx (hunk 1 above)
Comment (prakashsakari) [reply src/components/Category/Category.jsx]: thanks, updated!!

**Pull request merged**